pkg/rtc/transport.go

```diff
@@ -1254,32 +1255,31 @@ func (t *PCTransport) parseTrackMid(offer webrtc.SessionDescription, senders map
 	return nil
 }
 
-func (t *PCTransport) postEvent(event event) {
-	t.eventsQueue.Enqueue(t.handleEvent, event)
```

Comment:
this was still allocating... i thought because t was already on the heap it wouldn't but `t.handleEvent` seems to be treated same as capturing `t` in a closure like
```go
t.eventsQueue.Enqueue(func(e event) {
    t.handleEvent(e)
}, event)
```
so the interface `func(e event)` in the queue gets assigned something like 
```go
struct {
    t *PCTransport
    pc uint
}
``` 
but with a non-captring closure it's assigned the pc of the function. to avoid this we can pass the pointer to `t` in in the event instead